src/CRM/Graph.hs

```diff
@@ -10,10 +10,10 @@ newtype Graph a = Graph [(a, a)]
  It has as vertices the product of the set of vertices of the initial graph.
  It has as edge from `(a1, b1)` to `(a2, b2)` if and only if there is an edge
  from `a1` to `a2` and an edge from `b1` to `b2`
--}
 
--- >>> productGraph (Graph [('a', 'b')]) (Graph [('c', 'd')])
--- Graph [(('a','c'),('b','d'))]
+ >>> productGraph (Graph [('a', 'b')]) (Graph [('c', 'd')])
+ Graph [(('a','c'),('b','d'))]
+-}
```

Comment:
As I recall, doctests in multiline comments don't execute in VSCode (and potentially other HLS-powered editors too). That is unfortunate.

Did having it as multiline docstring + single line doctests cause any issues?

Reply:
so, execution in vscode works only with single-line style

doctests are recognised only when inside Haddock comments (i.e. the ones with a `|`).

For example

```haskell
{- | bla bla bla
-}

-- >>> example
-- result
```

would not work (meaning that the example is executable in vscode but won't be recognised by doctests).

I guess the only viable solution to get both working is using single-line Haddock comments

Reply:
I think that using single-line comments for everything is the best compromise.